src/components/Utilities/Modal.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, ReactNode } from 'react';
+
+interface ModalProps {
+  show: boolean;
+  children: ReactNode;
+  handleClose: (...args: any[]) => void;
+}
+
+const Modal: FC<ModalProps> = ({ show = false, children, handleClose }) => (
+  <div className={`modal${show ? ' show' : ''}`}>
```

Comment:
Some simple libraries like `clsx` or `classNames` are used to simplify classes adding